nix/lib/buildPinnacleRustConfig.nix

```diff
@@ -0,0 +1,14 @@
+{ craneLib, crateArgs, pinnacle-api }:
+{ src }:
+(craneLib.buildPackage (crateArgs // rec {
+  inherit src;
+  inherit (pinnacle-api) cargoArtifacts; # use pinnacle api cargo artifacts
+  pname = "pinnacle-config";
+  installPhaseCommand = ''
+    mkdir -p $out/share/pinnacle/config/
+    mkdir $out/bin
+    mv target/release/${pname} $out/bin/
+    mv metaconfig.toml $out/share/pinnacle/config/
+    ln $out/bin/${pname} $out/share/pinnacle/config/${pname};
```

Comment:
I do find this structure in the derivation a bit weird, with the config being in both `share/pinnacle/config/pinnacle-config` and `bin/pinnacle-config`. I would expected only next to the `metaconfig.toml`.